client/src/app/submission/submission.component.ts

```diff
@@ -41,6 +41,14 @@ export class SubmissionComponent implements OnInit, OnDestroy {
   manifest: Object;
   submissionErrors: Object[];
   selectedIndex: any = 0;
+  validationErrors: any;
+
+  // TODO: Give these a type
```

Comment:
yeah, you just need to put the MetadataDataSource as its type, right?

Reply:
Done for the data sources but the validationErrors will be given a type in error-summary PR